test/e2e/deploy_test.go

```diff
@@ -0,0 +1,126 @@
+package e2e_test
+
+import (
+	"os"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/pborman/uuid"
+
+	cli "github.com/mongodb/mongodb-atlas-kubernetes/test/e2e/utils"
+	. "github.com/onsi/gomega/gbytes"
+	"github.com/onsi/gomega/gexec"
+)
+
+var _ = Describe("Deploy simple cluster", func() {
+
+	It("Release sample all-in-one.yaml should work", func() {
+		By("Prepare namespaces")
+		namespaceUserResources := "e2e-" + uuid.NewRandom().String() // TODO
+		namespaceOperator := "mongodb-atlas-kubernetes-system"
+		session := cli.Execute("kubectl", "create", "namespace", namespaceUserResources)
+		Expect(session).ShouldNot(Say("created"))
+		session = cli.Execute("kubectl", "create", "namespace", namespaceOperator)
```

Comment:
my understanding is that we don't need to create this namespace manually - the K8s scripts already have this command

Reply:
deleted